sdk/monitor/azure-monitor-opentelemetry-exporter/tests/statsbeat/test_customer_statsbeat.py

```diff
@@ -34,6 +34,20 @@
 from azure.monitor.opentelemetry.exporter import AzureMonitorTraceExporter
 from azure.monitor.opentelemetry.exporter.statsbeat._state import _REQUESTS_MAP
 
```

Comment:
nit: on the customer statsbeat overall tests should likely a test where `APPLICATIONINSIGHTS_STATSBEAT_ENABLED_PREVIEW` is not set or set to "false" and assert that no metrics are collected.